### PR TITLE
Adds documentation for commands.onChanged

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary

Add details for the addition of commands.onChanged in 115.

### Related issues and pull requests

Addresses the dev-docs-needed requirement of [Bug 1801531](https://bugzilla.mozilla.org/show_bug.cgi?id=1801531) Implement browser.commands.onChanged.

Related BCD changes in [#19944](https://github.com/mdn/browser-compat-data/pull/19944)

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```js-nolint
+browser.commands.onChanged.addListener(listener)
+browser.commands.onChanged.removeListener(listener)
+browser.commands.onChanged.hasListener(listener)
+```
+
+Events have three functions:
+
+- `addListener(callback)`
```
Comment (willdurand) [inline files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md:24]: why would we use `callback` here but `listener` in the other two methods? that seems inconsistent.

**Review by willdurand — APPROVED** [review]

File: files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md (hunk 1 above)
Comment (Rob--W) [reply files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md:24]: That issue has happened before. Is there a similarly incorrect template that you have copied? Please fix that too (at least storage.onChanged and storageArea.onChanged, more if you can find them).

Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md:24]: This is a widespread issue (possibly in 69 events). I'll do a separate PR for the other, but the occurrence in commands are done.

Comment (Rob--W) [reply files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md]: SGTM, thanks

**Review by Rob--W — APPROVED** [review]
Will's review was sufficient here. Still, lgtm.

**Pull request merged**